docs/browser-acceptance-tests.md

```diff
@@ -189,10 +189,9 @@ Tests are organized into suites under the `test/browser_tests/cucumber/features`
 ## Performance testing
 
 To audit if the site complies with performance best practices and guidelines,
-run `gulp audit:perf`.
-
-The audit will run against
-[Google's Lighthouse](https://github.com/GoogleChrome/lighthouse).
+[Google's Lighthouse](https://github.com/GoogleChrome/lighthouse) can be run
+from Google Chrome by opening the developer console and going to the Lighthouse
+tab and run a performance audit.
```

Comment:
```suggestion
tab to run a performance audit.
```

Additionally, we can update this section once we can point to the nightly audit.